StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -11,10 +11,12 @@
 @_spi(STP) import StripeUICore
 import UIKit
 
-protocol PaymentSheetLinkAccountInfoProtocol {
-    var email: String { get }
-    var redactedPhoneNumber: String? { get }
-    var isRegistered: Bool { get }
+@_spi(STP) public protocol PaymentSheetLinkAccountInfoProtocol {
+    @_spi(STP) var email: String { get }
+    @_spi(STP) var redactedPhoneNumber: String? { get }
+    @_spi(STP) var isRegistered: Bool { get }
+    @_spi(STP) var sessionState: PaymentSheetLinkAccount.SessionState { get }
+    @_spi(STP) var consumerSessionClientSecret: String? { get }
```

Comment:
Discussed in Slack, but this change was intended to allow the new API method to `throw` when the account is not in a verified state. The final result in this PR throws additionally when the `consumerSessionClientSecret` is unset.

Reply:
They’ll match their containing declaration’s access level, so these properties are exposed as `public` by default. The compiler will enforce this, too.
<img width="569" height="89" alt="Xcode 2025-07-24 09 06 39" src="https://github.com/user-attachments/assets/b816128c-0e6c-4b26-a303-b45764f22f55" />